### PR TITLE
[Workers] Show WebAssembly.instantiate() in global scope

### PR DESCRIPTION
**Pull request by irvinebroque**

### CODE REVIEW

File: content/workers/wrangler/bundling.md [hunk 1]
```diff
@@ -33,10 +33,11 @@ This is also the basis of Wasm support with Wrangler. To use a Wasm module in a
 
 ```js
 import wasm from "./example.wasm"; // Where `example.wasm` is a file in your local directory
+const module = WebAssembly.instantiate(wasm); // Instantiate Wasm modules in global scope, not within the fetch() handler
 
 export default {
   fetch(request) {
-    const module = WebAssembly.instantiate(wasm);
+    // ...
```
Comment (kflansburg) [inline content/workers/wrangler/bundling.md]:
```suggestion
    const result = instance.exports.exported_func();
```

File: content/workers/wrangler/bundling.md [hunk 2]
```diff
@@ -33,10 +33,11 @@ This is also the basis of Wasm support with Wrangler. To use a Wasm module in a
 
 ```js
 import wasm from "./example.wasm"; // Where `example.wasm` is a file in your local directory
+const module = WebAssembly.instantiate(wasm); // Instantiate Wasm modules in global scope, not within the fetch() handler
```
Comment (kflansburg) [inline content/workers/wrangler/bundling.md]:
```suggestion
const instance = await WebAssembly.instantiate(wasm); // Instantiate Wasm modules in global scope, not within the fetch() handler
```

**Review by kflansburg — APPROVED** [review]

File: content/workers/wrangler/bundling.md (hunk 2 above)
Comment (deadlypants1973) [inline content/workers/wrangler/bundling.md]:
```suggestion
const instance = await WebAssembly.instantiate(wasm); // Instantiate Wasm modules in global scope, not within the `fetch()` handler
```

Comment (deadlypants1973) [reply content/workers/wrangler/bundling.md]: Adding to the above suggestion

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**